scipy/special/orthogonal.py

```diff
@@ -1386,6 +1388,26 @@ def gegenbauer(n, alpha, monic=False):
     :math:`[-1,1]` with weight function :math:`(1 - x^2)^{(\alpha -
     1/2)}`.
 
+    Examples
+    --------
+    >>> from scipy import special
+    >>> import matplotlib.pyplot as plt
+    >>> import numpy as np
+
+    >>> p = special.gegenbauer(3, 0.5, monic=False)
+    >>> p
+    poly1d([ 2.5,  0. , -1.5,  0. ])
+    >>> p(1)
+    1.0
+    >>> x = np.linspace(-3, 3, 400)
+    >>> y = p(x)
+
+    >>> plt.plot(x, y)
+    >>> plt.title("Gegenbauer (ultraspherical) polynomial of degree 3")
+    >>> plt.xlabel("x")
+    >>> plt.ylabel("G_3(x)")
```

Comment:
This is the recommended way to use matplotlib.

```suggestion
    >>> fig, ax = plt.subplots()
    >>> ax.plot(x, y)
    >>> ax.set_title("Gegenbauer (ultraspherical) polynomial of degree 3")
    >>> ax.set_xlabel("x")
    >>> ax.set_ylabel("G_3(x)")
```